curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/assignment-with-a-returned-value.english.md

```diff
@@ -9,9 +9,20 @@ forumTopicId: 16658
 ## Description
 <section id='description'>
 If you'll recall from our discussion of <a href="/learn/javascript-algorithms-and-data-structures/basic-javascript/storing-values-with-the-assignment-operator" target="_blank">Storing Values with the Assignment Operator</a>, everything to the right of the equal sign is resolved before the value is assigned. This means we can take the return value of a function and assign it to a variable.
-Assume we have pre-defined a function <code>sum</code> which adds two numbers together, then:
+numbers together, then:
```

Comment:
This reads as -
>  This means we can take the return value of a function and assign it to a variable. numbers together, then:

`"numbers together, then:"` is probably left over from the previous commit undo...

Reply:
What do you suggest?

Reply:
Unless I missed what you were going for, just removing "numbers together, then:" should fix it.

Reply:
@Manish-Giri I actually fixed this in a separate commit based on a similar request by @ojeytonwilliams.